foreshadow/foreshadow.py

```diff
@@ -66,12 +77,22 @@ def __init__(self, estimator=None, problem_type=None):
                 )
             )
         self.problem_type = problem_type
+        self.random_state = random_state
+        self.n_jobs = n_jobs
+
         self._X_preparer = DataPreparer(cache_manager=CacheManager())
+        self.configure_multiprocessing(self.n_jobs)
         self._y_preprarer = DataPreparer(
             cache_manager=CacheManager(),
             y_var=True,
             problem_type=self.problem_type,
         )
+        if estimator is not None and estimator_kwargs is not None:
```

Comment:
I wonder if this could be confusing to the user. The `estimator_kwargs` variable could be interpreted as the arguments you pass to the estimator. Should we rename it as `auto_estimator_kwargs` or `auto_estimator_configs`?

Reply:
Will do.